src/fprime_gds/executables/fprime_cli.py

```diff
@@ -14,7 +14,7 @@
 from typing import Callable, List, Union
 
 import argcomplete
-import pkg_resources
```

Comment:
`pkg_resources` was not available in Python3.12 and causing problems in CI. Replaced with `importlib` which is available in the std library for Python 3.8+ and is the recommended way of getting package versions